### ISSUE TITLE
Batch norm instead of instance norm

### ISSUE THREAD

**Issue by sunshower76** [Rudrabha/Wav2Lip#275]
Hello. Your work is really helpful!

I wanna modifiy your network. and I have questions.

1. Why do you use batchnormalization?
:  In common, GAN usually use instance normalization rather than batch normalization. Is there a special reason you use it?

2. Why adv_wt is only multiplied by the perceptual loss, not multiplied by the discriminator loss?
: I think this reason is that the model could learn the image to image mapping without considering audio features. so, the adversarial loss of the generator should be low. Is it right?

**Comment by Rudrabha** [comment]
1. We trained the Wav2Lip without the visual quality disc initially. This had batch normalization in the generator. We then added a visual quality disc once we had satisfactory sync quality. In the visual quality disc, we used [conv blocks](https://github.com/Rudrabha/Wav2Lip/blob/80c01f9fe1aaa07e19c1e3088744bb8500cb45fa/models/conv.py#L21) without normalization. So, we have not used batch norm in the GAN discriminator.
2. I don't fully understand this question. We have multiplied a weight to the adversarial loss here in this [line](https://github.com/Rudrabha/Wav2Lip/blob/80c01f9fe1aaa07e19c1e3088744bb8500cb45fa/hq_wav2lip_train.py#L239).
